nb/projects/belt_sprockets/examples.md

```diff
@@ -0,0 +1,136 @@
+```JavaScript
+import {
+  Pulley,
+  MXL,
+  DP40,
+  XL,
+  H,
+  T2_5,
+  T5,
+  T10,
+  AT5,
+  HTD_3mm,
+  HTD_5mm,
+  GTD2_5mm,
+  HTD_8mm,
+  GT2_2mm,
+  GT2_3mm,
+  GT2_5mm,
+} from './belt_sprockets.nb';
+```
+
+```JavaScript
+Pulley(MXL, 20).md('MXL').view();
```

Comment:
Do we want to add a .clean() to Pulley()? It seems like having the clean version is probably what most people would want

Reply:
Yes, I think that makes sense.